.github/workflows/stale_issue_checker.yml

```diff
@@ -0,0 +1,36 @@
+name: Stale Issue Checker
+permissions:
+  issues: write
+
+on:
+  schedule:
+    #       ┌───────────── minute (0 - 59)
+    #       │ ┌───────────── hour (0 - 23)
+    #       │ │ ┌───────────── day of the month (1 - 31)
+    #       │ │ │ ┌───────────── month (1 - 12 or JAN-DEC)
+    #       │ │ │ │ ┌───────────── day of the week (0 - 6 or SUN-SAT)
+    #       │ │ │ │ │
+    #       │ │ │ │ │
+    #       │ │ │ │ │
+    #       * * * * *
+    - cron: "0 0 * * *" # Runs once daily at 00:00 UTC
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Close stale issues
+        uses: actions/stale@v9
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          stale-issue-label: "stale"
+          any-of-issue-labels: "waiting for user response"
+          labels-to-add-when-unstale: "user responded"
+          labels-to-remove-when-stale: "user responded"
+          labels-to-remove-when-unstale: "waiting for user response"
```

Comment:
Are these really needed?

Reply:
This is required in order to reset the label. But other one is just for visibility 

```
labels-to-remove-when-unstale: "waiting for user response"
```